config/initializers/simple_form.rb

```diff
@@ -48,7 +48,7 @@
     # If you want to display the full error message for the attribute, you can
     # use the component :full_error, like:
     #
-    # b.use :full_error, wrap_with: { tag: :span, class: :error }
+    b.use :full_error, wrap_with: { tag: :span, class: :error }
```

Comment:
This seems like it potentially affects a ton of error messages.  Is there any way we can scope this down to just file type inputs?